rs/ic_os/fstrim_tool/BUILD.bazel

```diff
@@ -31,9 +31,8 @@ rust_library(
     aliases = ALIASES,
     crate_name = "ic_fstrim_tool",
     proc_macro_deps = MACRO_DEPENDENCIES,
-    visibility = [
-        "//rs:system-tests-pkg",
-    ],
+    # TODO: is this really needed?
```

Comment:
I see that [some crypto tests depend on it](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/tests/src/crypto/ic_crypto_fstrim_tool_test.rs?L32)